src/EnergyPlus/PurchasedAirManager.hh

```diff
@@ -175,8 +175,6 @@ namespace PurchasedAirManager {
         Real64 SupplyAirMassFlowRate;       // Supply air mass flow rate [kg/s]
         Real64 SupplyAirVolFlowRateStdRho;  // supply air volume flow using standard density [m3/s]
         // Intermediate results
-        Real64 FinalMixedAirTemp;     // Dry-bulb temperature of the mixed air, saved for system ventilation load reporting [C]
-        Real64 FinalMixedAirHumRat;   // Humidity ratio of the mixed air, saved for system ventilation load reporting [kgWater/kgDryAir]
         Real64 HtRecSenOutput;        // Sensible heating/cooling rate from heat recovery (<0 means cooling) [W]
```

Comment:
Removed them since I felt the new mixedairtemp and mixedairhumrat were doing the same thing?

Reply:
And it appears that  `FinalMixedAirTemp` and `FinalMixedAirHumRat` weren't being used anywhere, so find to replace them here.